_data/sites.json

```
@@ -495,6 +495,15 @@
         ]
     },
 
+    {
+        "name": "AwardWallet",
+        "url": "https://awardwallet.com/user/delete",
+        "difficulty": "easy",
+        "notes": "Site makes you add in your password and write a reason why you are deleting account",
+        "domains": [
+            "awardwallet.com"
+        ]
+    },
```

Add an empty line between entries:

```suggestion
    },

```